app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/produsent/api/MutationNySak.kt

```diff
@@ -104,7 +104,7 @@ internal class MutationNySak(
             nySak.erDuplikatAv(eksisterende) -> {
                 if (eksisterende.statusoppdateringIkkeRegistrert()) {
```

Comment:
jeg skjønner ikke helt implementasjonen av `statusoppdateringIkkeRegistrert`. 
Den er som følger: 
```kotlin
fun statusoppdateringIkkeRegistrert() =
            statusoppdateringer.any { it.idempotencyKey.startsWith(IdempotenceKey.initial()) }
``` 
For meg ser dette feil ut. 
I tillegg så er ikke sakId på `eksisterende` og `statusoppdateringHendelse` lik, som resulterer i en statusoppdatering på sakId som ikke finnes.